### PR TITLE
 Sub step about closing TOP in Windows and reminder to follow Linux instructions.

### LINKED ISSUE #28325: Installations: Clarify that copy pasting should happen in the VM
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Ever since a robust explanation of copy-pasting into the Terminal has been added to the Linux instructions the amount of VM users having issues with copy-pasting has increased by a lot. 

While this reduced the amount of people mistyping commands, it seems like the many sentences that tell learners that they should do everything in the VM after they set it up are not effective.

To complicate things, ever since TOP dropped Firefox support people are unaware that Xubuntu comes with a Web Browser. And perhaps ironically, it is a required part of the curriculum now to use something that it does not support.

There are potentially multiple avenues to explore that I can think of:
1. Separate VM instructions so they can explicitly call out opening Firefox inside the VM and copying from there.
2. In the setup instructions' Step 4 add a substep that asks the learner to close TOP in their Windows browser and to open it in the VM. Reiterate that from this point onwards they're supposed to follow Linux instructions while going through TOP.
3. Edit the copy-paste note box to include the VM exception.

I'm probably a fan of 2. because it feels the most natural for the flow of the instructions and it does not bloat the Chrome installation instructions further.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-installations

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by LAKSHYA201**
## Because

As mentioned in the issue , regardless of many sentences throughout the beginning that asked the users to do everything in VM have not been effective , so a specific highlighted sub step was required . There was also need of a reminder to be added that learners only follow the Linux instructions from now on.

## This PR

- A sub step 4.1 stating asking the user to close TOP in windows and open it in the VM along with a summarized explanations has been added .
- A "Remember" has been added to ask learners to follow only the linux instructions.

## Issue

Closes #28325 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by LAKSHYA201** [comment]
upon running npm run lint :project -- "path"  , there is no error but running lint:lesson produces an error :  Required heading structure [Expected: ### Lesson overview; Actual: #### Be mindful of the OS you are using].

is there something i should do about this or can do about this ?

**Comment by MaoShizhong** [comment]
> upon running npm run lint :project -- "path" , there is no error but running lint:lesson produces an error : Required heading structure [Expected: ### Lesson overview; Actual: #### Be mindful of the OS you are using].
> 
> is there something i should do about this or can do about this ? should i perhaps remove the #### and put a **Remember:**:-Be mindful of the OS you are using , if it would help ?

For this lint error, could you replace the first line with the following:
```md
<!-- TODO: Revisit lesson/heading structure to remove need to disable rules -->
<!-- markdownlint-disable MD024 TOP004 -->
```

The error's flagging because the current disable comment targets the wrong rule code, but we also need the TODO because this (and a few other installation lessons) have structures we need to rethink for accessibility reasons. They're lower priority for now, but we need the TODOs for when we get to them.

File: foundations/installations/installations.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you are already using **MacOS**, **Ubuntu**, or [an official flavor of Ubuntu
 
 #### Be mindful of the OS you are using
 
-We can only support the operating systems indicated above. Our instructions have been tested with MacOS, Ubuntu, and official flavors of Ubuntu. We do not recommend installing an OS that is only based on Ubuntu (like Mint, Pop!_OS, ElementaryOS, etc).
+We can only support the operating systems indicated above. Our instructions have been tested with MacOS, Ubuntu, and official flavors of Ubuntu. We do not recommend installing an OS that is only based on Ubuntu (like Mint, Pop!\_OS, ElementaryOS, etc).
```
Comment (xandora) [inline foundations/installations/installations.md:11]: Not sure we need to escape this underscore? The markdown parser on the site appears to handling it correctly already.

File: foundations/installations/installations.md [hunk 2]
```diff
@@ -196,10 +196,16 @@ After `sudo apt upgrade` runs for a while you will be asked whether you want to
 
 ### Step 4: Understand your new VM
 
+#### Step 4.1: Close TOP in the Windows browser and open the website in your VM
```
Comment (xandora) [inline foundations/installations/installations.md:199]: Seems reasonable enough to me. Hopefully we'll put a dent in the amount of questions this concept generates on the Discord with this.

**Review by xandora — CHANGES_REQUESTED** [review]
Overall, looks good to me! Just wondering about the escaped underscore...

File: foundations/installations/installations.md (hunk 1 above)
Comment (LAKSHYA201) [reply foundations/installations/installations.md]: The backslash seemed to have crept up due to Some sort of Auto Formatting , i disabled Format on save setting for this one , and it seems fine now.

**Review by xandora — APPROVED** [review]
I don't have any further issues with this.

**Comment by LAKSHYA201** [comment]
Hi @MaoShizhong , sorry for abruption but i have solved the issue and wanted to remind you that the PR is ready to be merged, just in case it slipped the maintainers' radars.

**Pull request merged**